src/background.js

```diff
@@ -137,12 +139,13 @@ const rollout = {
     await stateManager.setState("UIDisabled");
     await stateManager.rememberDoorhangerDecision("UIDisabled");
     await stateManager.rememberDoorhangerPingSent();
+    // eslint-disable-next-line no-undef
     browser.experiments.heuristics.sendHeuristicsPing("doorhangerDecline", results);
```

Comment:
I think that this is a bug that is pre-existing. I'm not sure how to fix it.

Reply:
@mythmon Confirmed. I'll fix this before signing v10